FirebasePerformance/Sources/FPRDataUtils.m

```diff
@@ -116,9 +116,10 @@
     if (![pathSeparator isEqual:truncationCharacter]) {
       NSRange rangeOfTruncation = [truncatedURLString rangeOfString:pathSeparator
                                                             options:NSBackwardsSearch];
-      truncatedURLString = [URLString substringToIndex:rangeOfTruncation.location];
+      if (rangeOfTruncation.location != NSNotFound) {
+        truncatedURLString = [URLString substringToIndex:rangeOfTruncation.location];
```

Comment:
Do we need to add some log message in else branch. If so, any suggestion is welcomed.

Reply:
I think it is fine without any log message.